designs.js

```diff
@@ -13,15 +13,13 @@ const submit = document.getElementById('MakeCatalog');
 )
 
 
-var i = 0;
+//var i = 0;
```

Comment:
Could you please remove the line? It's considered to be a bad practice to keep commented out blocks of code. If you don't need some code just remove it before commit.